docs/developingWorkflows/developing.rst

```diff
@@ -154,9 +167,14 @@ parse command line options for a Toil script. For example::
             return "Hello, world!, here's a message: %s" % self.message
 
     if __name__=="__main__":
-        parser = Job.Runner.getDefaultArgumentParser()
```

Comment:
Don't get rid of this and replace it with the default options.  Read the instructions above about what's it's telling the user.